src/main/java/com/github/seratch/jslack/api/model/File.java

```diff
@@ -94,4 +108,36 @@ public void setPublic(boolean isPublic) {
     private List<String> pinnedTo;
     private List<Reaction> reactions;
     private Integer commentsCount;
+
+    private Shares shares;
+
+    @Data
+    public static class Shares {
+
+        /**
+         * The key of the Map: channel ID
+         */
+        @SerializedName("public")
```

Comment:
To begin with, I added `public` and `private`. If there are other patterns (im?), I will add them in further releases.